new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiChangeAdminCredentials.java

```diff
@@ -0,0 +1,528 @@
+
+// Test to change the WebLogic credentials secret of a domain custom resource that uses model-in-image.
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Test to patch the model-in-image image to change WebLogic admin credentials secret")
+@IntegrationTest
+class ItMiiChangeAdminCredentials implements LoggedTest {
+
+  private static String domainNamespace = null;
+  private static String domainUid = "domain1";
+  private static ConditionFactory withStandardRetryPolicy = null;
+
+  private static String adminServerPodName = String.format("%s-%s", domainUid, ADMIN_SERVER_NAME_BASE);
+  private static String managedServerPrefix = String.format("%s-%s", domainUid, MANAGED_SERVER_NAME_BASE);
+  private static int replicaCount = 2;
+
+  /**
+   * Perform the following initialization for all the tests in this class:
+   * 1) set up the necessary namespaces for the operator and one domain,
+   * 2) install the operator in the first namespace, and
+   * 3) create a domain in the second namespace using the pre-created basic MII image.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+        .and().with().pollInterval(10, SECONDS)
+        .atMost(6, MINUTES).await();
+
+    // get namespaces 
+    assertNotNull(namespaces.get(0), String.format("Namespace namespaces.get(0) is null"));
+    String opNamespace = namespaces.get(0);
+
+    assertNotNull(namespaces.get(1), String.format("Namespace namespaces.get(1) is null"));
+    domainNamespace = namespaces.get(1);
+
+    // install the operator
+    logger.info("Install an operator in namespace {0}, managing namespace {1}",
+        opNamespace, domainNamespace); 
+    installAndVerifyOperator(opNamespace, domainNamespace);
+   
+    // create a domain resource
+    logger.info("Create model-in-image domain {0} in namespace {1}, and wait until it comes up",
+        domainUid, domainNamespace); 
+    createAndVerifyMiiDomain();
+  }
+  
+  /**
+   * Test patching a running model-in-image domain with a new WebLogic credentials secret。
+   * The test performs two patching operations to the domain spec：
+   * 1） Change the domain spec's webLogicCredentialsSecret；
+   * 2） Change the domain spec's domainRestartVersion to trigger a rolling restart of the server pods。
+   * The test verifies the following：
+   * 1） the domain spec's webLogicCredentialsSecret and restartVersion are updated；
+   * 2） the server pods are recreated by checking each pod's creationTimestamp before and after patching；
+   * 3） the server pods' weblogic.domainRestartVersion label is updated；
+   * 4） the new credentials are valid and can be used to access WebLogic RESTful Management Services。
```

Comment:
Remove  the index (1) (2) (3) (4) from javadoc

Reply:
done.